client/users/user-profile.tsx

```diff
@@ -378,11 +401,11 @@ function SummaryPage({
         ))}
       </TotalGamesSection>
 
-      <SectionOverline>Latest games</SectionOverline>
+      <SectionOverline>{t('users.profile.latestGames', 'Latest games')}</SectionOverline>
       <MiniMatchHistory forUserId={user.id} games={matchHistory} />
 
-      <SectionOverline>Achievements</SectionOverline>
-      <EmptyListText>Nothing to see here</EmptyListText>
+      <SectionOverline>{t('users.profile.achievements', 'Achievements')}</SectionOverline>
+      <EmptyListText>{t('users.profile.noAchievements', 'Nothing to see here')}</EmptyListText>
```

Comment:
Really feel like we could pull this string into some common place for general empty states, it's all over the app